agentpluginapi/i_linux_agent_command_builder.py

```diff
@@ -47,6 +47,12 @@ def build_download_command(self, download_options: LinuxDownloadOptions):
         :param download_options: Options needed for the command to be built
         """
 
+    @abc.abstractmethod
+    def build_chmod_command(self):
```

Comment:
If command builder is a utility to build command/bash statements then we should be specific. If there's another way to change permissions we'll have to add it to the interface, because it will be a different bash statement.